ERCS/erc-7549.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
```suggestion
---
eip: 7583
```
Assigning next sequential EIP/ERC/RIP number. 7549 has already been issued.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename and the Eth Magicians discussion thread.

Reply:
The ERC number in the filename and the Eth Magicians discussion thread has been updated to 7583.